eksrollup/lib/k8s.py

```diff
@@ -127,16 +147,29 @@ def cordon_node(node_name):
 
     # create an instance of the API class
     k8s_api = client.CoreV1Api()
-    logger.info("Cordoning k8s node {}...".format(node_name))
-    try:
-        api_call_body = client.V1Node(spec=client.V1NodeSpec(unschedulable=True))
-        if not app_config['DRY_RUN']:
-            k8s_api.patch_node(node_name, api_call_body)
-        else:
-            k8s_api.patch_node(node_name, api_call_body, dry_run=True)
-        logger.info("Node cordoned")
-    except ApiException as e:
-        logger.info("Exception when calling CoreV1Api->patch_node: {}".format(e))
+    if cordon:
```

Comment:
I would suggest you simplify this block since the if and else are pretty much duplicated. 
you can do something like:

```
unschedulabe = false
if cordon:
         logger.info("Cordoning k8s node {}...".format(node_name))
         unschedulabe = true
else:
         logger.info("uncordoning k8s node {}...".format(node_name))
         unschedulabe = false

 try:
            api_call_body = client.V1Node(spec=client.V1NodeSpec(unschedulable))
            if not app_config['DRY_RUN']:
                k8s_api.patch_node(node_name, api_call_body)
            else:
                k8s_api.patch_node(node_name, api_call_body, dry_run=True)
            logger.info("Node uncordoned")
        except ApiException as e:
            logger.info("Exception when calling CoreV1Api->patch_node: {}".format(e))
    .
    .
    .
```

or you can extract this functionality in a private function and just invoke it here.

Reply:
great point 💯  - this is repeating for sure! I've done some edits, let me know what you think